### PR TITLE
Fix CheckStyle issues in net/GlowNetworkServer.java

### PR DESCRIPTION
**Pull request by Pr0methean**

### CODE REVIEW

File: src/main/java/net/glowstone/net/GlowNetworkServer.java [hunk 1]
```diff
@@ -8,13 +8,20 @@
 /**
  * Represents a network server.
  *
- * Modified implementation of {@link com.flowpowered.network.NetworkServer}.
+ * <p>Modified implementation of {@link com.flowpowered.network.NetworkServer}.</p>
```
Comment (mastercoms) [inline src/main/java/net/glowstone/net/GlowNetworkServer.java]: Don't use a closing p tag.

Comment (Pr0methean) [reply src/main/java/net/glowstone/net/GlowNetworkServer.java]: Fixed. I'll go remove them all on the master branch as well.

Comment (mastercoms) [reply src/main/java/net/glowstone/net/GlowNetworkServer.java]: Sorry for missing those.

Comment (Pr0methean) [reply src/main/java/net/glowstone/net/GlowNetworkServer.java]: No, I mean in the branch my other style-fix PRs will be split from. I'm not sure there are any pre-existing closing &lt;/p>'s.

**Pull request merged**